lxd/storage/drivers/driver_powerflex.go

```diff
@@ -32,6 +34,9 @@ type powerflex struct {
 	// Holds the low level HTTP client for the PowerFlex API.
 	// Use powerflex.client() to retrieve the client struct.
 	httpClient *powerFlexClient
+
+	// Holds the SDC GUID of this specific host.
```

Comment:
This could do with a similar comment to above that indicates not to use this var directly,but instead use  getHostGUID

Reply:
Fixed.